lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -953,277 +974,236 @@
 
     // Intl.NumberFormat, Number.prototype.toLocaleString
     var NumberFormat = (function () {
+        if (InitType !== "Intl" && InitType !== "Number") {
+            return;
+        }
         // Keep these "enums" in sync with lib/Runtime/PlatformAgnostic/Intl.h
-        const NumberFormatStyle = setPrototype({
+        const NumberFormatStyle = {
             DEFAULT: 0, // "decimal" is the default
             DECIMAL: 0, // Intl.NumberFormat(locale, { style: "decimal" }); // aka in our code as "number"
             PERCENT: 1, // Intl.NumberFormat(locale, { style: "percent" });
             CURRENCY: 2, // Intl.NumberFormat(locale, { style: "currency", ... });
-        }, null);
-        const NumberFormatCurrencyDisplay = setPrototype({
+        };
+        const NumberFormatCurrencyDisplay = {
             DEFAULT: 0, // "symbol" is the default
             SYMBOL: 0, // Intl.NumberFormat(locale, { style: "currency", currencyDisplay: "symbol" }); // e.g. "$" or "US$" depeding on locale
             CODE: 1, // Intl.NumberFormat(locale, { style: "currency", currencyDisplay: "code" }); // e.g. "USD"
             NAME: 2, // Intl.NumberFormat(locale, { style: "currency", currencyDisplay: "name" }); // e.g. "US dollar"
-        }, null);
-
-        if (InitType === "Intl" || InitType === "Number") {
-            function InitializeNumberFormat(numberFormat, localeList, options) {
-                if (typeof numberFormat != "object") {
-                    platform.raiseNeedObject();
-                }
-
-                if (callInstanceFunc(ObjectInstanceHasOwnProperty, numberFormat, "__initializedIntlObject") && numberFormat.__initializedIntlObject) {
-                    platform.raiseObjectIsAlreadyInitialized("NumberFormat", "NumberFormat");
-                }
-
-                numberFormat.__initializedIntlObject = true;
+        };
 
-                // Extract options
-                if (typeof options === "undefined") {
-                    options = setPrototype({}, null);
-                } else {
-                    options = Internal.ToObject(options);
-                }
+        const SetNumberFormatDigitOptions = function (nf, options, mnfdDefault, mxfdDefault) {
+            const mnid = GetNumberOption(options, "minimumIntegerDigits", 1, 21, 1);
```

Comment:
>mnid [](start = 18, length = 4)

in general I feel like we're abbreviating too much at the cost of readability. mnid -> minid or minID and mxid -> maxid or maxID would be better IMO.

Reply:
This is only to keep in line with the spec naming.